numpy/core/tests/test_multiarray.py

```diff
@@ -7196,8 +7196,30 @@ def test_view_assign(self):
         assert_(not arr_wb.ctypes.data == 0)
         arr_wb[:] = 100
         assert_equal(arr, -100)
-        
-        
+
+
+class TestArange(object):
+    def test_infinite(self):
+        assert_raises_regex(
+            ValueError, "size exceeded",
+            np.arange, 0, np.inf
+        )
```

Comment:
It would be more conventional to keep the closing parenthesis on the previous line.